ors-api/src/main/java/org/heigit/ors/api/responses/export/topojson/Network.java

```diff
@@ -0,0 +1,19 @@
+
+package org.heigit.ors.api.responses.export.topojson;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.io.Serializable;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Getter
+@Builder
+public class Network implements Serializable {
+    @JsonProperty("network")
+    private Layer network;
```

Comment:
I find it a bit confusing that the class and the field are both named network but the field is of type Layer.

Reply:
renamed